.github/work/linters.yml

```diff
@@ -0,0 +1,62 @@
+name: Linters
```

Comment:
- [ ] It looks like your linters creation could improve, try to add .github/workflows (check the folder name), and inside the workflows folder insert the linter.yml file. 🐶

Reply:
Done!